### ISSUE TITLE
Error: estimator is not supported

### ISSUE THREAD

**Issue by M-Younus** [TeamHG-Memex/eli5#376]
Got `Error: estimator is not supported` .
Using tensorflow.keras
python=3.7.4
tensorflow=2.1.0
eli5=0.10.1

**Comment by teabolt** [comment]
Hi. Unfortunately `tensorflow.keras` is currently not supported. As in https://github.com/TeamHG-Memex/eli5/blob/017c738f8dcf3e31346de49a390835ffafad3f1b/eli5/keras/explain_prediction.py#L8 only `keras` models are supported. For now one solution is to convert a `tensorflow.keras` model into a `keras` model.

**Comment by shivammavihs** [comment]
@teabolt are you aware of any alternative to eli5 which supports keras and does the job?
